src/components/Chips/__stories__/chips.stories.mdx

```diff
@@ -150,6 +154,55 @@ e.g. `<Chips label="GRASS_GREEN" color={Chips.colors.GRASS_GREEN} />`
   </Story>
 </Canvas>
 
+### Chips on colored backgrounds
+
+When a chip appears on a background color identical to its color, use `showBorder` prop in order to add a distinctive white border.
+<Canvas>
+  <Story name="On color">
+    <Flex style={{ width: "100%", height: 300 }} align={Flex.align.STRETCH} justify={Flex.justify.CENTER}>
```

Comment:
Let's change height to be smaller? No need to use 300px here, could be 100px, e.g.